hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/DeleteMetadataActor.scala

```diff
@@ -38,9 +42,15 @@ class DeleteMetadataActor(metadataDeletionConfig: ActiveMetadataDeletionConfig,
       workflowIdsForMetadataDeletionFuture onComplete {
         case Success(workflowIds) =>
           if (workflowIds.length < metadataDeletionConfig.batchSize) {
-            numOfWorkflowsToDeleteMetadataMetricActor ! NumberOfWorkflowsToDeleteMetadataMetricValue(workflowIds.length.toLong)
+            numOfWorkflowsToDeleteMetadataMetricActor ! MetricValue(workflowIds.length.toLong)
           } else {
-            numOfWorkflowsToDeleteMetadataMetricActor ! CalculateNumberOfWorkflowsToDeleteMetadataMetricValue(currentTimestampMinusDelay)
+            numOfWorkflowsToDeleteMetadataMetricActor ! CalculateMetricValue {
+              () =>
+                countRootWorkflowSummaryEntriesByArchiveStatusAndOlderThanTimestamp(
```

Comment:
TOL: this might be easier to read if you extract this into a `def`? eg 
```
else {
  def deleteMetricCalculation() = ???
  numOfWorkflowsToDeleteMetadataMetricActor ! CalculateMetricValue(deleteMetricCalculation)
}
```